### PR TITLE
feat(stark-ui): implement Pagination component. Integrate pagination in Table component

### PR DESCRIPTION
**Pull request by christophercr**
ISSUES CLOSED: #539 #514 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NationalBankBelgium/stark/blob/master/CONTRIBUTING.md#-commit-message-guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #539 #514 

## What is the new behavior?
Pagination component available in the Stark UI package and integrated in the Table component.

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: showcase/src/assets/translations/fr.json [hunk 1]
```diff
@@ -141,6 +141,14 @@
         "LOGOUT_CUSTOM_ICON": "Bouton logout - Icône custom",
         "TITLE": "Stark logout"
       },
+      "PAGINATION": {
```
Comment (catlabs) [inline showcase/src/assets/translations/fr.json]: "PAGINATION": {
        "ADVANCED_CONFIGURATION": "Pagination avancée",
        "COMPACT_CONFIGURATION": "Pagination en mode compact",
        "EMITTED_PAGINATE_EVENT": "Pagination avec paginate event",
        "EXTENDED_CONFIGURATION": "Pagination selection de page",
        "SIMPLE_CONFIGURATION": "Pagination simple",
        "TITLE": "Stark Pagination"
      },

Comment (christophercr) [reply showcase/src/assets/translations/fr.json]: Thanks!

**Comment by christophercr** [comment]
Remarks solved. And rebased ;)

**Comment by christophercr** [comment]
Added pagination example files :)

**Pull request merged**